clevercsv/console/commands/standardize.py

```diff
@@ -265,10 +263,10 @@ def _in_place(
         return 2
 
     def _to_stdout(
-        self, path: StrPath, dialect: SimpleDialect, encoding: Optional[str], target_encoding: Optional[str]
+        self, path: StrPath, dialect: SimpleDialect, encoding: Optional[str]
     ) -> int:
         stream = io.StringIO(newline="")
```

Comment:
Target-encoding has no effect on sdtout because StringIO save unicode in memory. 
If having non-utf8 on stdout can be relevant, maybe we can considere to use BytesIO ?

Reply:
Good catch! I think we can leave that for a separate pull request if you like.